### PR TITLE
1140 ping pong

### PR DESCRIPTION
**Pull request by eslopfer**
Add readme file with the instructions

### CODE REVIEW

File: subjects/pingpong/README.md [hunk 1]
```diff
@@ -0,0 +1,22 @@
+### Instructions
```
Comment (Hamzaelkhatri) [inline subjects/pingpong/README.md:1]: Add name of exercises 
> ## ping-pong

Comment (eslopfer) [reply subjects/pingpong/README.md:1]: Thank you

**Review by Hamzaelkhatri — APPROVED** [review]

File: subjects/pingpong/README.md [hunk 2]
```diff
@@ -0,0 +1,24 @@
+## pingpong
+
+### Instructions
+
+Write a program that checks if all numbers up to a given number are odd or even.
+The numbers will be between 0 and 100.
+
+- If the number is odd then it prints the number and `ping` as in the example below.
```
Comment (MSilva95) [inline subjects/pingpong/README.md:8]: In the usage you are saying that the number 2 for example is odd when 2 is an even number ...
This is wrong here you should be saying that 
"If the number is even then it prints the number and `ping` as in the example below."

Comment (eslopfer) [reply subjects/pingpong/README.md:8]: I'll correct that. Thank you.

**Review by MSilva95 — APPROVED** [review]

**Pull request merged**